examples/advanced/cifar10/cifar10-sim/figs/plot_tensorboard_events.py

```diff
@@ -30,7 +30,7 @@
 
 # 4.1 Central vs. FedAvg
 experiments = {
-    "cifar10_central": {"tag": "val_acc_local_model"},
+    "cifar10_central": {"tag": "val_acc_local_model", "alpha": 0.0},
```

Comment:
What does "alpha" mean?
What does it mean with centralized setting?

Reply:
alpha=0.0 splitting of the dataset, i.e. centralized. See https://github.com/NVIDIA/NVFlare/blob/dd9d2da59321d902e935083b97915e3e58cf5ed0/examples/advanced/cifar10/cifar10-sim/set_alpha.py#L46